src/MapViewController.swift

```diff
@@ -128,6 +141,49 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate, TG
     }
   }
 
+  public func display(route: OTRRoutingResult) throws {
+    //TODO: We eventually should support N number of routes.
+    if let routeMarker = currentRouteMarker {
+      //We don't throw if the remove fails here because we want to silently replace the route
+      currentRouteMarker = nil
+      markerRemove(routeMarker)
+    }
+    let routeLeg = route.legs[0]
```

Comment:
So do we want to support it here as a single polyline? Or just punt on it for now.